### PR TITLE
Add 2.1 arm32v7 aspnetcore-runtime

### LINKED ISSUE #480: Add arm32v7 2.1-aspnetcore-runtime image
I [recently created](https://github.com/aspnet/Universe/issues/554) a linux-arm version of the aspnetcore shared runtime. I would like to add a linux-arm image containing this runtime, similar to the bionic/arm32v7 image we have for just the netcore runtime.

### PR DESCRIPTION
**Pull request by sdmaclea**
Fixes #480 

@MichaelSimons @natemcmaster PTAL

### CODE REVIEW

File: 2.1/aspnetcore-runtime/bionic/arm32v7/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM microsoft/dotnet:2.1-runtime-deps-bionic-arm32v7
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+        curl \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install .NET Core
+ENV ASPNETCORE_VERSION 2.1.0-preview3-32175
```
Comment (MichaelSimons) [inline 2.1/aspnetcore-runtime/bionic/arm32v7/Dockerfile]: Are rc1 builds available?  It would be nice to align the builds with amd64.

**Comment by MichaelSimons** [comment]
Thanks Steve for the PR.  A few more changes are necessary in order to merge.  I apologize there isn't more doc on this.

1. Can you make this PR against the `nightly` branch?  These will get merged into the official repo when RC1 is released.  
2. You will need to modify the [manifest.json](https://github.com/dotnet/dotnet-docker/blob/master/manifest.json) to include entries for these new.  This is what the builds and CI operate off of.  Here is what I think would be needed off the top of my head:
At https://github.com/dotnet/dotnet-docker/blob/master/manifest.json#L400 add:
```
            {
              "architecture": "arm",
              "dockerfile": "2.1/aspnetcore-runtime/stretch-slim/arm32v7",
              "os": "linux",
              "tags": {
                "2.1.0-rc1-aspnetcore-runtime-stretch-slim-arm32v7": {},
                "2.1-aspnetcore-runtime-stretch-slim-arm32v7": {}
              },
              "variant": "armv7"
            },
```
At https://github.com/dotnet/dotnet-docker/blob/master/manifest.json#L534, add:
```
        {
          "platforms": [
            {
              "architecture": "arm",
              "dockerfile": "2.1/aspnetcore-runtime/bionic/arm32v7",
              "os": "linux",
              "tags": {
                "2.1.0-rc1-aspnetcore-runtime-bionic-arm32v7": {},
                "2.1-aspnetcore-runtime-bionic-arm32v7": {}
              }
            }
          ]
        },
```
3. The unit tests should be updated to cover these new images.  I am hopeful you can duplicate the [2.1 arm test cases](https://github.com/dotnet/dotnet-docker/blob/master/test/Microsoft.DotNet.Docker.Tests/ImageTests.cs#L36) and add `, IsWeb = true`
4. To update the readme, the https://github.com/dotnet/dotnet-docker/blob/master/scripts/TagsDocumentationTemplate.md should be updated and then you can run https://github.com/dotnet/dotnet-docker/blob/master/scripts/Get-TagsDocumentation.ps1 if you'd like.  There are too many tags and links to keep straight :)

File: 2.1/aspnetcore-runtime/bionic/arm32v7/Dockerfile (hunk 1 above)
Comment (sdmaclea) [reply 2.1/aspnetcore-runtime/bionic/arm32v7/Dockerfile]: I will match nightly version.

**Comment by sdmaclea** [comment]
I will close this and reopen when I have a nightly PR ready.

**Pull request closed without merging**